flink-ml-lib/src/main/java/org/apache/flink/ml/common/param/linear/HasLearningRate.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.common.param.linear;
+
+import org.apache.flink.ml.param.DoubleParam;
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.param.ParamValidators;
+import org.apache.flink.ml.param.WithParams;
+
+/** Interface for the shared learning rate param. */
+public interface HasLearningRate<T> extends WithParams<T> {
```

Comment:
I didn't find this learning rate parameter in the logistic regression wiki [1] or in Spark [2]. Could you explain why we need this learning rate parameter but Spark's logistic regression does not?

[1] https://en.wikipedia.org/wiki/Logistic_regression

Reply:
Learning rate is concept in machine learning, please refer to Sklearn[1] or Tensorflow[2].

Spark used stepSize. But it is not consistent with other ML libraries.

[1] https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html
[2] https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/SGD